src/index.d.ts

```diff
@@ -7,6 +7,7 @@ export interface ExplosionProps {
     y: number
   };
   explosionSpeed?: number;
+  size?: number;
```

Comment:
```suggestion
  maxSize?: number;
```

would be more accurate, don't you think ?